src/client/apps/edit/components/header/yoast/index.tsx

```diff
@@ -0,0 +1,249 @@
+import { Box, color, Flex, space } from "@artsy/palette"
+import { avantgarde } from "@artsy/reaction/dist/Assets/Fonts"
+import Icon from "@artsy/reaction/dist/Components/Icon"
+import {
+  Input,
+  StyledInput,
+  Title,
+} from "@artsy/reaction/dist/Components/Input"
+import { ArticleData } from "@artsy/reaction/dist/Components/Publishing/Typings"
+import { StaticCollapse } from "@artsy/reaction/dist/Components/StaticCollapse"
+import React, { Component } from "react"
+import { connect } from "react-redux"
+import styled from "styled-components"
+import {
+  App as YoastApp,
+  SnippetPreview as YoastSnippetPreview,
+} from "yoastseo"
+import { setYoastKeyword } from "../../../../../actions/edit/editActions.js"
+
+interface Props {
+  article: ArticleData
+  yoastKeyword: string
+  setYoastKeywordAction: (e: any) => void
+}
+
+interface State {
+  isOpen: boolean
+}
+
+export class Yoast extends Component<Props, State> {
+  private snippetPreview
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isOpen: false,
+    }
+  }
+
+  componentDidMount() {
+    this.snippetPreview = new YoastSnippetPreview({
+      targetElement: document.getElementById("yoast-snippet"),
+    })
+    const app = new YoastApp({
```

Comment:
do we need to do any cleanup of app when this component gets unmounted?
If yes, we should keep a reference and cleanup in `componentDidUnmount`